osu.Game/Screens/Edit/Setup/ResourcesSection.cs

```diff
@@ -70,7 +73,7 @@ private void load()
                 audioTrackTextBox = new FileChooserLabelledTextBox
                 {
                     Label = "Audio Track",
-                    Current = { Value = Beatmap.Value.Metadata.AudioFile ?? "Click to select a track" },
+                    Current = { Value = working.Value.Metadata.AudioFile ?? "Click to select a track" },
```

Comment:
Why were all these changed from going through the `EditorBeatmap` to the `IBindable<WorkingBeatmap>`? Seems contrary to all other changes in this PR.

Reply:
Anything touching raw database elements I opted to use the working beatmap (ie. things outside of the beatmap .osu parsing stuff itself) as it seems like a better ground truth. Right now, the former code only works because it was referencing the same metadata - wasn't sure this is how things are going to be in the future so felt it safer to use working.

Am fine with reverting if you prefer (will probably need reconsideration with database refactoring either way).

Reply:
Well `MetadataSection` above which also refers to "database elements" goes through the `EditorBeatmap`. Let's revert.

Reply:
Oh this file actually does database operations... Unfortunate.

Reply:
Yeah, needs double checking that using the `EditorBeatmap` will actually work.